static/js/resize.js

```diff
@@ -277,6 +277,9 @@ exports.handler = function () {
     popovers.hide_all();
     exports.resize_page_components();
 
+    // big hammer
+    current_msg_list.view.rerender_the_whole_thing();
```

Comment:
Can you test whether replacing it with `condense.condense_and_collapse($("tr.message_row"));` still fixes the bug?

Reply:
Nope, that doesn't seem to work